x-pack/plugins/fleet/server/services/epm/packages/get_bulk_assets.ts

```diff
@@ -33,11 +33,9 @@ const getKibanaLinkForESAsset = (type: ElasticsearchAssetType, id: string): stri
     case 'data_stream_ilm_policy':
       return `/app/management/data/index_lifecycle_management/policies/edit/${id}`;
     case 'transform':
-      // TODO: Confirm link for transforms
-      return '';
+      return `/app/management/data/transform?_a=(transform:(queryText:${id}))`;
     case 'ml_model':
-      // TODO: Confirm link for ml models
-      return '';
+      return `/app/ml/trained_models?_a=(trained_models:(queryText:'model_id:(${id})'))`;
```

Comment:
Same for models - would be nice to add the description, if it exists.

Reply:
I was directed to keep it consistent in that file, for now. If maintainability becomes an issue then we can bring it up.